src/function/unit/toBest.js

```diff
@@ -0,0 +1,46 @@
+import { factory } from '../../utils/factory.js'
+
+const name = 'toBest'
+const dependencies = ['typed']
+
+export const createToBest = /* #__PURE__ */ factory(name, dependencies, ({ typed }) => {
+  /**
+   * Converts a unit to the most appropriate display unit.
+   * When no preferred units are provided, the function automatically find the best prefix.
+   * When preferred units are provided, it converts to
+   * the unit that gives a value closest to 1.
+   *
+   * Syntax:
+   *
+   *    math.toBest()
+   *    math.toBest(unitList)
+   *    math.toBest(unitList, options)
+   *
+   * Where:
+   * - `unitList` is an optional array of preferred target units as string or Unit.
+   * - `options` is an optional object with options, formed as follows:
+   * - `offset`: number | BigNumber
+   *
+   * Examples:
+   *
+   *    math.unit(0.05, 'm').toBest(['cm', 'mm'])                         // Unit 500 cm
```

Comment:
I see [one test is failing](https://github.com/josdejong/mathjs/actions/runs/16047418655/job/45372895732?pr=3484), and this is because the example says the outcome here will be `500 cm` but actually the outcome is `5 cm`, so the calculation is correct but the comment is incorrect. Can you fix the comment?